Fedoraware/TeamFortress2/TeamFortress2/Features/Auto/AutoUber/AutoUber.cpp

```diff
@@ -36,10 +36,14 @@ int BulletDangerValue(CBaseEntity* pPatient)
 		default: { continue; }
 		}
 
-		if (HAS_CONDITION(player, TFCond_Bonked))
+		if (HAS_CONDITION(player, (TFCond_Bonked | TFCond_Cloaked))) //is enemy cloaked/bonked?
```

Comment:
This will check if the player is Bonked AND Cloaked at the same time

Reply:
yes

Reply:
Issue is that they will never be both bonked and cloaked

> yes